apps/site/components/SlideOver/SlideOverUsage.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+  SlideOver,
+  SlideOverHeader,
+  SlideOverProps,
+  useFadeEffect,
+  useUI,
+} from '@faststore/ui'
+
+export type SlideOverUsageProps = {
+  isOpen: SlideOverProps['isOpen']
+  size: SlideOverProps['size']
+  direction: SlideOverProps['direction']
+}
+
+const SlideOverUsage = (props: SlideOverUsageProps) => {
```

Comment:
Please, use the destructuring in these props to follow the pattern

```suggestion
const SlideOverUsage = ({ isOpen, size, direction }: SlideOverUsageProps) => {
```